File: source/source/cs12/CustomControl/FlatTextBox.cs
using System;
using System.Web.UI.WebControls;

namespace CustomControl
{
	/// <summary>
	/// Summary description for FlatTextBox.
	/// </summary>
	public class FlatTextBox : TextBox
	{
		public FlatTextBox() // 생성자
		{
			// TextBox 스타일 설정하기
			this.Style["border"] = "solid 1px black";
			this.Font.Name = "verdana";
			this.ForeColor = System.Drawing.Color.Navy;
			this.Font.Size = FontUnit.XSmall;
			this.BackColor = System.Drawing.Color.DeepPink;
		}
	}
}

File: source/source/vb12/CustomControl/FlatTextBox.vb
Imports System.Web.UI.WebControls

Public Class FlatTextBox
    Inherits TextBox

    Public Sub New()    ' 생성자
        ' TextBox 스타일 설정하기
        Me.Style.Item("border") = "solid 1px black"
        Me.Font.Name = "verdana"
        Me.ForeColor = System.Drawing.Color.Navy
        Me.Font.Size = FontUnit.XSmall
        Me.BackColor = System.Drawing.Color.DeepPink
    End Sub


End Class
